### PR TITLE
Draw polaroid behind training fish

### PR DESCRIPTION
**Pull request by madelynkasula**
Screen Shot 2019-11-18 at 2 17 23 PM

**Note:** The background of the polaroid was lighter in Figma, but the sock wasn't rendering well on that color, so Mark gave me the okay to try a darker background for the polaroid.

### CODE REVIEW

File: src/demo/renderer.js [hunk 1]
```diff
@@ -440,23 +463,6 @@ const drawPredictBot = state => {
   ctx.drawImage(botImg, botX, botY);
 };
 
-// Draw frame in the center of the screen.
```
Comment (madelynkasula) [inline src/demo/renderer.js]: unrelated clean-up: deleted this method as it wasn't used anywhere

File: src/demo/renderer.js [hunk 2]
```diff
@@ -547,75 +558,49 @@ export const clearCanvas = canvas => {
...
 
   const canvasCtx = getState().canvas.getContext('2d');
   canvasCtx.globalAlpha = amount;
-  canvasCtx.fillStyle = overlayColour;
-  DrawFilledRect(0, 0, constants.canvasWidth, constants.canvasHeight);
+  DrawRect(0, 0, constants.canvasWidth, constants.canvasHeight, overlayColour);
   canvasCtx.globalAlpha = 1;
-}
-
-const drawRoundedFrame = (
```
Comment (madelynkasula) [inline src/demo/renderer.js]: unrelated clean-up: deleted this method as it wasn't used anywhere

File: src/demo/renderer.js [hunk 3]
```diff
@@ -352,7 +351,15 @@ const drawMovingFish = state => {
...
 
-    drawSingleFish(fish, x, y, ctx, 1, drawPrediction);
+    drawSingleFish(
+      fish,
+      x,
+      y,
+      ctx,
+      1,
+      drawPrediction,
+      state.currentMode === Modes.Training
```
Comment (bethanyaconnor) [inline src/demo/renderer.js]: Can you comment on what this line is signaling? Just something like `// drawPolaroid`
